src/claps/claps.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Controller, Post, Param, UseGuards, Request } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import {
+  ApiBearerAuth,
+  ApiTags,
+  ApiParam,
+  ApiCreatedResponse,
+} from '@nestjs/swagger';
+
+import { Article } from '@src/articles/domain/article';
+import { ClapsService } from '@src/claps/claps.service';
+@ApiTags('Claps')
+@Controller({
+  path: 'articles',
```

Comment:
Is it intentional? otherwise I think it should be 'claps'.

Reply:
Yeah it is because I thought the following endpoint might be good
"/api/v1/articles/[id]/clap"
Let me Know if I need to modify it